src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Patient's priority for treatment in ORGANice.
+ * The higher the priority, the more likely they will be matched with donor's organs.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPriority(String)}
+ */
+public class Priority {
+
+    public static final String MESSAGE_CONSTRAINTS = "Priority can only be high, medium or low"
+            + "and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+
+    public static final String PRIORITY_HIGH = "high";
+    public static final String PRIORITY_MEDIUM = "medium";
+    public static final String PRIORITY_LOW = "low";
+
+
+    public final String value;
+
+    /**
+     * Constructs a {@code Priority}.
+     *
+     * @param priority A valid priority.
+     */
+    public Priority(String priority) {
+        requireNonNull(priority);
+        checkArgument(isValidPriority(priority), MESSAGE_CONSTRAINTS);
+        value = priority;
```

Comment:
`value = priority.toLowerCase()` so we can save all datas as lower cases

Reply:
Made a change